samples/drivers/rtc/ds1307/prj.conf

```diff
@@ -0,0 +1,12 @@
+# Enable Peripheral
+CONFIG_I2C=y
+CONFIG_RTC=y
+CONFIG_RTC_DS1307=y
```

Comment:
`CONFIG_RTC_DS1307=y` is automatically selected if the ds1307 exists in the devicetree with status "okay", so it can be removed